### ISSUE TITLE
`Cannot find module '../lightningcss.linux-x64-gnu.node'`

### ISSUE THREAD

**Issue by jacob-orbiit** [parcel-bundler/lightningcss#567]
This appears to have broken on `1.21.5` (I can't repro it on `1.21.3`)

Related: https://github.com/parcel-bundler/lightningcss/issues/335

**Comment by devongovett** [comment]
Probably https://github.com/npm/cli/issues/4828

**Comment by silverwind** [comment]
Fix your lockfile: `rm -rf node_modules package-lock.json && npm i`.

**Comment by Eric-Arellano** [comment]
Thanks! @silverwind's suggestion worked for me to fix GitHub Actions erroring when running `parcel build` with NPM.

**Comment by Gurucaara** [comment]
> Fix your lockfile: `rm -rf node_modules package-lock.json && npm i`.
Thanks! @silverwind's suggestion worked for me to fix vercel deployment

**Comment by jacob-orbiit** [comment]
Heads up: you should never delete your lockfile.

Also, this package is obsolete¹: esbuild handles css-modules now: https://esbuild.github.io/content-types/#local-css

¹ Thanks for it though!

**Comment by thomashagstrom** [comment]
How is this closed? Deleting lock file does not help

**Comment by atul24112001** [comment]
I was facing the same issue, the problem in my case was expo@52.0.21 and nativewind@4.1.23 is using the same package which is lightningcss but with different version. You can check in your project by running
`npm list lightningcss` 
This command will list all the packages using lightningcss with its version if you see any version mismatch that's the problem you can solve this by running
`npm dedupe`
and then
`npm rebuild lightningcss` 
This  solved the issue for me hope works for you as well.

**Comment by maribox** [comment]
This did not solve the issue for me, both versions installed are the same:

```
❯ npm run dev

> frontend@0.0.1 dev
> LIBC=glibc vite dev

failed to load config from /home/fed/Coding/Rust/AudioComparison/frontend/vite.config.ts
error when starting dev server:
Error: Cannot find module '../lightningcss.linux-x64-musl.node'
Require stack:
- /home/fed/Coding/Rust/AudioComparison/frontend/node_modules/lightningcss/node/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Module._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:141:16)
    at Object.<anonymous> (/home/fed/Coding/Rust/AudioComparison/frontend/node_modules/lightningcss/node/index.js:21:22)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1689:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32)
❯ npm list lightningcss


frontend@0.0.1 /home/fed/Coding/Rust/AudioComparison/frontend
├─┬ @tailwindcss/vite@4.0.9
│ └── lightningcss@1.29.1
└─┬ vite@6.2.0
  └── lightningcss@1.29.1 deduped
```

**Comment by emran-binhasan** [comment]
facing the issue suddenly after trying to use tailwind in vite

**Comment by emran-binhasan** [comment]
The issue was solved after instaling Microsoft redistributable package. download from here https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by dnwjn** [comment]
> I was facing the same issue, the problem in my case was expo@52.0.21 and nativewind@4.1.23 is using the same package which is lightningcss but with different version. You can check in your project by running
> 
> `npm list lightningcss`
> 
> This command will list all the packages using lightningcss with its version if you see any version mismatch that's the problem you can solve this by running
> 
> `npm dedupe` and then `npm rebuild lightningcss`
> 
> This solved the issue for me hope works for you as well.

I just want to thank you for this! I was having issues in my GitHub workflow. No matter what I tried, I kept getting issues related to "sharp" or "lightningcss". Finally I came across your comment, and that fixed it for me!

**Comment by tbanj** [comment]
read my comment at this link, it will help to solve this issue
https://github.com/expo/eas-cli/issues/2727#issuecomment-2779228557

**Comment by AbirDoesFrontend** [comment]
Thanks @silverwind It worked for me❤️

**Comment by rsshilli** [comment]
For those of you who understand supply chain attacks and don't want to delete your package.lock file, this worked for me:
```
npm update lightningcss
```

It'll update just the lightningcss package without updating the others.

**Comment by karanx712** [comment]
- Problem: Node version conflict on setup (running 24.x.x when 22.17.0 was required).
- Fix: Used nvm use 22.17.0, deleted node_modules and package-lock.json, and ran npm install.
- Status: Running smoothly.
